src/components/query/QualityAndOrderRow.tsx

```diff
@@ -104,14 +104,18 @@ export class QualityAndOrderRow extends PureComponent<Props> {
 
   render() {
     const { query } = this.props;
+    let currQualities = qualities;
+    if (query.queryType === QueryType.PropertyInterpolated) {
+      currQualities = qualities.slice(1);
+    }
```

Comment:
nit:

add this to the top of the file underneath where `const qualities` is defined so it's only created once.

```
const interpolatedPropertyQualities: Array<SelectableValue<SiteWiseQuality>> = qualities.slice(1);
```

and change this here to

```suggestion
    const currQualities = query.queryType === QueryType.PropertyInterpolated ? interpolatedPropertyQualities : qualities;
```

Reply:
done